### DISCUSSION TITLE
Default activity

### DISCUSSION THREAD

**Discussion by jens-2305** [kimai/kimai#3010] [General]
Good morning,

we would find it helpful if a activity could be selected as the default activity for time recording so that is .is it possible to include this as a setting for the future?

Thank you very much

**Comment by kevinpapst** [comment]
There is a feature request already. Please search it and vote for it (by using the 👍 on the initial post)

**Comment by jens-2305** [comment]
Thanks but I only found "Default project/activity per user? #2639" but the conversation their is blocked

**Comment by kevinpapst** [comment]
Have a look at those: 
https://github.com/kevinpapst/kimai2/issues/699
https://github.com/kevinpapst/kimai2/issues/403

You can search for "default" in the issues section to find other ones
